presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkConfig.java

```diff
@@ -191,4 +194,19 @@ public PrestoSparkConfig setSplitAssignmentBatchSize(int splitAssignmentBatchSiz
         this.splitAssignmentBatchSize = splitAssignmentBatchSize;
         return this;
     }
+
+    @DecimalMin("0.0")
+    @DecimalMax("1.0")
+    public double getMemoryRevokingThreshold()
+    {
+        return memoryRevokingThreshold;
+    }
+
+    @Config("spark.memory-revoking-threshold")
+    @ConfigDescription("Revoke memory when memory pool is filled over threshold")
```

Comment:
This property is copied from `FeaturesConfig`. Why not reuse `experimental.memory-revoking-threshold`?  It is a config to control spilling threshold for entire memory pool.   We want a session property that can only be used for Presto on Spark.